pageserver/src/layered_repository.rs

```diff
@@ -112,6 +112,8 @@ impl Repository for LayeredRepository {
     ) -> Result<Arc<dyn Timeline>> {
         let mut timelines = self.timelines.lock().unwrap();
 
+        assert_eq!(start_lsn, start_lsn.aligned());
```

Comment:
nit: maybe add is_aligned method to lsn?

Reply:
Good spot. Please consider the following names:

* `fn align(&self) -> Self`
* `fn aligned(&self) -> bool` (apparently, we lean towards omitting `is_` prefixes).